Caddyfile

```diff
@@ -0,0 +1,10 @@
+{$DOMAIN} {
```

Comment:
I know you mentioned of wanting to add a Caddy tag, but this project would definitely not be the place for it IMO, this has no need for Caddy since it's just semantics at that point, especially for something that is just acting as a receiver. Just my two cents, if you can provide a good reason to include another web instance that is already here aside from it just being Caddy, I would prefer to remove it, sorry!!

Reply:
See above, please!